app/services/preserved_object_handler.rb

```diff
@@ -191,7 +189,11 @@ def moab_validation_errors
     moab_errors
   end
 
-  def create_db_objects(status, ran_moab_validation=false)
+  def ran_moab_validation?
+    @ran_moab_validation
```

Comment:
👍

Reply:
ah, good question, hadn't thought about that explicitly... but i think what's happening is that if an instance var hasn't been set yet, and it gets read, `nil` will get returned, which is "falsey", so essentially yeah, it returns `false` for our use case.

but that's a little unclear, so i'll change the method to do `@ran_moab_validation ||= false`.  which'll explicitly init `@ran_moab_validation` to false if it's still unset, or leave it alone if it's already there.  hopefully things read a bit clearer with that change.

Reply:
good catch, @SaravShah and @jmartin-sul !